project.godot

```diff
@@ -31,6 +36,7 @@ _global_script_classes=[ {
 } ]
 _global_script_class_icons={
```

Comment:
A little unclear on what makes something an icon

Reply:
Absolutely no idea. I THINK this has to do with the icon the IDE shows for a given node type; we define some custom nodes that can be instantiated through the IDE